keps/sig-api-machinery/1929-built-in-default/README.md

```diff
@@ -414,48 +416,64 @@ Both these cases will be handled properly, here's an example:
 type Object struct {
      // This field has an invalid zero-value, empty string is not a
      // valid string.
-     // +default=default-name
-     Name string
+     // +default="default-name"
+     Name string `json:"name,omitempty"`
 
-     // This field is defaulted to 0 with no defaulter.
+     // This field would be defaulted to 0 without a default.
```

Comment:
Why would it be default to 0 without a default? What about non-omitempty cases in general?